dist/js/jsflap.js

```diff
@@ -81,7 +81,7 @@
                             } else {
```

Comment:
I just realized that you are editing the generated file and not the source files, you have to use `npm install` and `gulp build` or `watch` to actually add code like you did in `src/Board/Board.ts`.

Any of the angular stuff or UI logic can go in `src/jsflap.js`, but everything like in the Board class should go in the respective file.